src/data/builds/barbarian/2001_barb_min_100.ts

```diff
@@ -31,7 +31,7 @@ export const build: Build = {
         "15": "*3",
       },
       comment:
-        "Max Big Pick, Copper Collector, Tool Proficiency, and Motherlode Miner.\n*1 Excess points can go into Firmly Grasp it or Absolute Unit for more Str.\n*2 Points can also go into HP\n*3 If you need EXP, you can level Tempestuous Emotions",
+        "Max Big Pick, Copper Collector, Tool Proficiency, and Motherlode Miner.\n*1 Excess points can go into Firmly Grasp it or Absolute Unit for more Str.\n*2 Points can also go into HP\n*3 If you need EXP, you can level Tempestuous Emotions. Do not use this after you reach the highest available tool.",
```

Comment:
Same point about the XP, also with the `.` it's not clear that it's a comment about `Tempestuous Emotions`